Doc/reference/datamodel.rst

```diff
@@ -1632,9 +1633,14 @@ Basic customization
    Called after the instance has been created (by :meth:`__new__`), but before
    it is returned to the caller.  The arguments are those passed to the
    class constructor expression.  If a base class has an :meth:`__init__`
-   method, the derived class's :meth:`__init__` method, if any, must explicitly
+   method, the derived class's :meth:`__init__` method, if any, may need to
    call it to ensure proper initialization of the base class part of the
    instance; for example: ``super().__init__([args...])``.
+   Any :meth:`__init__` method that is overridden is not implicitly called.
```

Comment:
This feels redundant with the previous sentence + that’s not specific to `__init__`